app/models/ncr/work_order.rb

```diff
@@ -120,11 +122,21 @@ def current_approver_email_address
         self.currently_awaiting_approvers.first.email_address
       elsif self.approving_official
         self.approving_official.email_address
+      elsif self.emergency and self.approvers.empty?
+        EMERGENCY_APPROVER_EMAIL
```

Comment:
Right...see the latter part of https://github.com/18F/C2/pull/681/files#r41535674.